packages/@aws-cdk/cx-api/lib/cxapi.ts

```diff
@@ -31,6 +31,13 @@ export const DISABLE_VERSION_REPORTING = 'aws:cdk:disable-version-reporting';
 export const DISABLE_ASSET_STAGING_CONTEXT = 'aws:cdk:disable-asset-staging';
 
 /**
+ * Tag metadata key.
+ */
+export const TAGS_METADATA_KEY = 'aws:cdk:stack-tags';
```

Comment:
This needs to be in `metadata.ts` and rename to `STACK_TAGS_METADATA_KEY`

Reply:
0e3c101